examples/advanced/streaming/jobs/README.md

```diff
@@ -0,0 +1,102 @@
+# Streaming API Example Jobs
+
+The `file_streaming` job can be used to benchmark/test file transferring performance of NVFlare ObjectStreamer API.
+
```

Comment:
This example is good, but I am thinking do we want to put this here?
Does user benefit from running this job?
Or if we want user to use, we should instead write an Admin Command?

Reply:
The example shows how to transfer a file from client to server using streaming. This is to replace a QA tool we deprecated. It may be beneficial to users so it's placed under example. We have another example using Flare API but it's hard to do on multiple machines so this one is in normal job format.